contracts/test/ERC20toCW20PointerTest.js

```diff
@@ -86,7 +86,7 @@ describe("ERC20 to CW20 Pointer", function () {
             });
 
             describe("transfer()", function () {
-                it("should transfer", async function () {
```

Comment:
That seems wrong, will now only run that tests, seems like a leftover @philipsu522